allofplos/plos_corpus.py

```diff
@@ -229,19 +229,17 @@ def repo_download(dois, tempdir, ignore_existing=True, plos_network=False):
         existing_articles = [filename_to_doi(file) for file in listdir_nohidden(tempdir)]
         dois = set(dois) - set(existing_articles)
 
-    if dois:
-        for doi in tqdm(sorted(dois)):
-            url = URL_TMP.format(doi)
-            articleXML = et.parse(url)
-            article_path = doi_to_path(doi, directory=tempdir)
-            # create new local XML files
-            if ignore_existing is False or ignore_existing and os.path.isfile(article_path) is False:
-                with open(article_path, 'w') as file:
-                    file.write(et.tostring(articleXML, method='xml', encoding='unicode'))
-                if not plos_network:
-                    time.sleep(1)
-    else:
-        pass
+    for doi in tqdm(sorted(dois), disable=len(dois)==0):
+        url = URL_TMP.format(doi)
+        articleXML = et.parse(url)
+        article_path = doi_to_path(doi, directory=tempdir)
+        # create new local XML files
+        if ignore_existing is False or ignore_existing and os.path.isfile(article_path) is False:
+            with open(article_path, 'w') as file:
+                file.write(et.tostring(articleXML, method='xml', encoding='unicode'))
```

Comment:
file is not a keyword but is a built-in type, so it is not advisable to use it as a name. Change it to file_ or fh or something else.
Also, I think that "not ignore_existing" is more easy to read than "ignore_existing is False".

Reply:
fixed in https://github.com/PLOS/allofplos/pull/59/commits/55751a87486cf60090a3c7d11dab68008d6d4644